heka/meta/heka.yml

```diff
@@ -459,10 +468,12 @@ aggregator:
       max_buffer_size: 1048576
       max_file_size: 524288
       full_action: drop
+  {%- endif %}
 {%- endif %}
+{%- if ceilometer_collector.influxdb_host is defined or ceilometer_collector.elasticsearch_host is defined or ceilometer_collector.amqp_host is defined %}
```

Comment:
Please remove this.

@ityaptin and I discussed this on Slack, and we decided to make `heka_alarming.py:grains_for_mine` more robust.